realsense/plugin.cpp

```diff
@@ -27,6 +27,8 @@ class realsense : public plugin {
         , _m_rgb_depth{sb->get_writer<rgb_depth_type>("rgb_depth")}
         , realsense_cam{ILLIXR::getenv_or("REALSENSE_CAM", "auto")}
         {      
+            cam_data.iteration = -1;
```

Comment:
Is this part the fix and the rest is just renaming variables and refactoring types?

Reply:
Yup!